### PR TITLE
fix few misspell and one small cleaning

### PR DESCRIPTION
**Pull request by dvrkps**

### CODE REVIEW

File: autorest/sender.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func (sf SenderFunc) Do(r *http.Request) (*http.Response, error) {
 	return sf(r)
 }
 
-// SendDecorator takes and possibily decorates, by wrapping, a Sender. Decorators may affect the
+// SendDecorator takes and possibility decorates, by wrapping, a Sender. Decorators may affect the
```
Comment (jhendrixMSFT) [inline autorest/sender.go]: I think this word is supposed to be `possibly`.

**Comment by jhendrixMSFT** [comment]
One little tweak, and if you can rebase on top of tip should be good to go.

File: autorest/sender.go (hunk 1 above)
Comment (dvrkps) [reply autorest/sender.go]: Done.

**Comment by vladbarosan** [comment]
@dvrkps I updated the branch for you with latest master. @jhendrixMSFT I guess we will do a patch release for this. Do you want a changelog although I would say we can go without that since there is no functional change.

**Comment by jhendrixMSFT** [comment]
@vladbarosan agreed no changelog update required.

**Pull request merged**